runner.py

```diff
@@ -9,13 +9,16 @@
 player = Player(player_name, 100)
 
 
-decision1 = input ("You ponder whether or not you should approach the ship. Do you want to approach the ship? Y/N: ").upper() 
+decision1 = input ("\nYou ponder whether or not you should approach the ship. Do you want to approach the ship? Y/N: ").upper() 
 if decision1 == "Y":
-    print("You approach the ship and are greeted by a tall, aristocratic man.")
-    character = Character("Samuel", 100)
-    decision2 = input("The man says: Hello, odd individual who is suspiciously close to my covert operation! What's your name? (tell him? Y/N:) ")
+    print("\nYou approach the ship and are greeted by a tall, aristocratic man.")
+    character = Character("Samuel", 150)
+    decision2 = input("\nThe man says: Hello, odd individual who is suspiciously close to my covert operation! What's your name? (tell him? Y/N:) ").upper()
     if decision2 == "Y":
-        character.talk_to_player(self, player, "x")
-    
+        character.talk_to_player(player, "x")
+        Item.use_item(player)
```

Comment:
This is throwing an error because the item doesn't exist. It's created in the character method talk to player, but the scope of those items created in that method don't reach this line of code. So a couple options here! You can run the use item inside the characters method or return the item from that method. I'll make some suggestions!